flask_project/campaign_manager/reminderRedisClient.py

```diff
@@ -0,0 +1,11 @@
+from redis import Redis
```

Comment:
I dont see redis in the requirements.txt

Reply:
From the Kartoza guidelines: module file name: concise (preferably single word) lower case, underscore separated e.g. utilities.py.

reminderRedisClient.py -> reminder_redis_client.py -> redis.py